xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -401,6 +403,39 @@ rebuild_master:
       <para>OBS assumes the configuration file is placed in your target branch
       and not in other branches or forks.</para>
```

Comment:
Sorry, I didn't realize this before. I'd also remove this line, it's not 100% true anymore, and this way we only rely on the new Configure File Location section for all the related information.

```suggestion
```